alacritty/Cargo.toml

```diff
@@ -35,6 +35,7 @@ libc = "0.2"
 unicode-width = "0.1"
 bitflags = "1"
 dirs = "3.0.1"
+lazy_static = "1.4.0"
```

Comment:
Not a fan of using lazy_static here, is this really necessary?

Reply:
Guess I could add it to the Logger struct instead.